pkg/controller/nexus/resource/route.go

```diff
@@ -19,8 +19,9 @@ package resource
 
 import (
 	"fmt"
+
 	"github.com/m88i/nexus-operator/pkg/apis/apps/v1alpha1"
-	"github.com/openshift/api/route/v1"
+	v1 "github.com/openshift/api/route/v1"
```

Comment:
Same thing here. Let me know your thoughts!

```suggestion
	routev1 "github.com/openshift/api/route/v1"
```

Reply:
Same comment as above.